src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -3433,7 +3433,7 @@ internal int GetCodePage(SqlCollation collation, TdsParserStateObject stateObj)
                         case 0x827:     // Mapping Non-supported Lithuanian code page to supported Lithuanian.
                             try
                             {
-                                codePage = Locale.GetCodePageForLcid(0x427);
+                                codePage = new CultureInfo(0x427).TextInfo.ANSICodePage;
```

Comment:
I based the changes in this file on the [reference source](https://github.com/Microsoft/referencesource/blob/cf5e702dcbf0fd1f5f5df75b3166345591a5e98a/System.Data/System/Data/SqlClient/TdsParser.cs#L3655-L3718).